src/sentry/models/organization.py

```diff
@@ -623,6 +623,14 @@ def get_scopes(self, role: Role) -> FrozenSet[str]:
             scopes.discard("alerts:write")
         return frozenset(scopes)
 
+    def get_teams_with_org_role(self, role):
+        from sentry.models.team import Team
+
+        if role:
+            return Team.objects.filter(org_role=role, organization=self)
+
+        return Team.objects.filter(~Q(org_role=None), organization=self)
```

Comment:
Django also has `.exclude()` if you want to write negated conditions without using `Q()`.